homeassistant/components/config/entity_registry.py

```diff
@@ -0,0 +1,55 @@
+"""HTTP views to interact with the entity registry."""
+import voluptuous as vol
+
+from homeassistant.core import callback
+from homeassistant.components.http import HomeAssistantView
+from homeassistant.components.http.data_validator import RequestDataValidator
+from homeassistant.helpers.entity_registry import async_get_registry
+
+
+async def async_setup(hass):
+    """Enable the Entity Registry views."""
+    hass.http.register_view(ConfigManagerEntityView)
+    return True
+
+
+class ConfigManagerEntityView(HomeAssistantView):
+    """View to interact with an entity registry entry."""
+
+    url = '/api/config/entity_registry/{entity_id}'
+    name = 'api:config:entity_registry:entity'
+
+    async def get(self, request, entity_id):
+        """Get the entity registry settings for an entity."""
+        hass = request.app['hass']
+        registry = await async_get_registry(hass)
```

Comment:
`hass.helpers.entity_register.async_get_register()`

Reply:
It's so long, not sure if I am a fan